exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExportUtils.java

```diff
@@ -79,7 +79,11 @@ static MetricDescriptor createMetricDescriptor(View view, String projectId) {
     String viewName = view.getName().asString();
     // Name format refers to
     // cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.metricDescriptors/create
-    builder.setName(String.format("projects/%s", projectId));
+    builder.setName(
+        // DO_NOT_SUBMIT: Should this also be "custom.googleapis.com/opencensus/%s"
```

Comment:
You're right, all custom metric names need to be under "custom.googleapis.com/" domain:
(https://cloud.google.com/monitoring/custom-metrics/creating-metrics)

So the whole name should follow the format: projects/my-project-id/metricDescriptors/custom.googleapis.com/opencensus/viewName

Reply:
Done.